roles/confluent.variables/vars/main.yml

```diff
@@ -9,10 +9,10 @@ zookeeper_default_group: confluent
 zookeeper_default_log_dir: /var/log/kafka
 zookeeper:
   server_start_file: "{{ binary_base_path }}/bin/zookeeper-server-start"
-  config_file: "{{ archive_config_base_path if installation_method == 'archive' else '' }}/etc/kafka/zookeeper.properties"
+  config_file: "{{ archive_config_base_path if installation_method == 'archive' else '' }}{{ zookeeper_config_prefix }}/zookeeper.properties"
```

Comment:
Ah I see, yall don't want `/etc/kafka/` there. Just curious, what do you plan to set `zookeeper_config_prefix` to?

Reply:
Correct. The objective here is to allow the configuration prefix as well as the specific component configuration path to be fully configurable. 

The default for `zookeeper_config_prefix` resolves to `/etc/kafka` as defined [here](https://github.com/confluentinc/cp-ansible/pull/639/files#diff-f8574416f3f69d6fd31fd81dc86ab06f89febbc73f6ae943a902ce3e4f63fa22R196).

Reply:
Yes the default looks fine, I meant in your inventory file what do you plan to set `zookeeper_config_prefix` to?

Reply:
> Yes the default looks fine, I meant in your inventory file what do you plan to set `zookeeper_config_prefix` to?

In this case, an empty string... The final location is computed as:

```
zookeeper:
  config_file: "{{ archive_config_base_path if installation_method == 'archive' else '' }}{{ zookeeper_config_prefix }}/zookeeper.properties"
```

In this case, the goal is to put it into some arbitrary path, such as `/opt/some/path/property/files`, so we'd set `archive_config_base_path: /opt/some/path/property/files` and `zookeeper_config_prefix: ""`, for a final computed value of `/opt/some/path/property/files/zookeeper.properties`.

However, some components also place property files into another subdirectory, e.g. `/opt/some/path/property/files/control-center`, and in that case, `control_center_config_prefix` would be set to `"control-center"`, for a final computer path of `/opt/some/path/property/files/control-center/control-center-production.properties` ... (all property files are typically placed at the base prefix at a minimum - e.g. `archive_config_base_path`)...